web/controllers/github_api/http_client.ex

```diff
@@ -82,4 +97,21 @@ defmodule Dwylbot.GithubAPI.HTTPClient do
     %{"pull_request" => pr}
   end
 
+  def get_data(token, payload, "list_pull_requests") do
+    "#{@github_root}/repos/#{payload["repository"]["full_name"]}/pulls"
+    |> HTTPoison.get!(header(token), [])
+    |> Map.fetch!(:body)
+    |> PP.parse!
+    |> Enum.map(fn(pr) ->
+      Map.merge(
+        get_data(
+          token,
+          %{"pull_request" => %{"url" => pr["url"]}},
+          "pull_request"
+        ),
+        get_data(token, %{"issue" => %{"url" => pr["issue_url"]}}, "issue")
```

Comment:
This is nice, but I think it would be easier to read if the formatting of lines `107->111` was the same as `112`